test/unit/sql/data-types.test.js

```diff
@@ -233,6 +233,30 @@ describe(Support.getTestDialectTeaser('SQL'), () => {
       });
     });
 
+    describe('TIME', () => {
+      testsql('TIME', DataTypes.TIME, {
+        ibmi: 'TIME',
+        postgres: 'TIME',
+        mssql: 'TIME',
+        mariadb: 'TIME',
+        mysql: 'TIME',
+        db2: 'TIME',
+        sqlite: 'TIME',
+        snowflake: 'TIME',
+      });
+
+      testsql('TIME.ZONED', DataTypes.TIME.ZONED, {
```

Comment:
With the current implementation, I think `DataTypes.TIME({ zoned: true })` doesn't work. Can you add a test for that?